cosmwasm/contracts/crosschain-swaps/src/contract.rs

```diff
@@ -1,4 +1,4 @@
-#[cfg(not(feature = "imported"))]
+#[cfg(not(feature = "library"))]
```

Comment:
I noticed you changed these. In the Cargo.toml files we're not defining the "library" feature but defining it as "imported" instead (https://github.com/osmosis-labs/osmosis/blob/main/cosmwasm/contracts/crosschain-swaps/Cargo.toml#L29). This is because many things are already called "library" and it can lead to confiusion.

This is not standard at the moment, but I have an issue to change it: https://github.com/CosmWasm/cw-template/issues/140

Reply:
Oh. I see